spec/examples/exist-upload

```diff
@@ -292,7 +304,7 @@ Usage:
       console.log("target exists:", existsAndCanOpenCollection)
     }
 
-    return uploadFileOrFolder(db, source, target, parsed)
+    return uploadFileOrFolder(db, rc, source, target, parsed)
   }
   catch (error) {
     if (error.name !== 'YError') {
```

Comment:
is `yerror` provided by `yargs` ? do we need an import here, typo?

Reply:
Correct `YError` is the error type from yargs. As it is a string I am not sure what to import here.